components/capi/include/pluralrules.h

```diff
@@ -0,0 +1,46 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+#pragma once
+#include <stdint.h>
+#include <stddef.h>
+#include <stdbool.h>
+#include "provider.h"
+#include "locale.h"
+
+// opaque
+typedef struct PluralRules PluralRules; 
+
+typedef struct {
+    PluralRules* rules;
+    bool success;
+} CreatePluralRulesResult;
+
+typedef enum {
+    PluralRuleType_Cardinal,
+    PluralRuleType_Ordinal
+} PluralRuleType;
```

Comment:
```suggestion
typedef enum {
    ICU4X_PluralRuleType_Cardinal,
    ICU4X_PluralRuleType_Ordinal
} ICU4X_PluralRuleType;
```

Thoughts?

Reply:
The convention for c enums that I'm used to is all caps, but that can be hard on the eyes. Maybe Icu4x_ as a prefix?

Reply:
> Maybe Icu4x_ as a prefix?

Mmmm, I don't really like the aesthetics of mixed-case on "ICU4X", especially since the first character "I" is confusable with lowercase letter "l".  I'd rather it be all uppercase or all lowercase.